src/Microsoft.DotNet.CodeAnalysis.net45/Microsoft.DotNet.CodeAnalysis.net45.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
That will issues a build warning as the analyzers target 4.5.2. I can change this to be 4.5.2 instead... or change the project name.